tests/test_core.py

```diff
@@ -7462,7 +7452,7 @@ def test_stack_overflow_check(self):
     self.emcc_args = args + ['-s', 'ASSERTIONS=1']
     self.do_run(open(path_from_root('tests', 'stack_overflow.cpp')).read(), 'Stack overflow! Attempted to allocate')
 
-  @no_wasm_backend('Wasm backend emits non-trapping float-to-int conversion')
+  @no_wasm_backend('BINARYEN_TRAP_MODE not meaningfull with llvm backend')
```

Comment:
Previous comment I think is useful too, it gives the "why". Either we should add that context to this message, or move it into a comment.

typo: meaningful

Reply:
Technically I don't think that why is correct.  The new why is simply that the option under test doesn't exist.. it doesn't matter what the default is.. the option simply doesn't work.   I updated the comment to include this though..